app/Models/UserFieldValue.php

```diff
@@ -22,6 +22,14 @@ class UserFieldValue extends Model
 
     public static $rules = [];
 
+    /**
+     * Return related field's name along with field values
+     */
+    public function getNameAttribute(): string
+    {
+        return $this->field->name;
```

Comment:
Check that field is valid first, you can have field values that are arbitrary without a field necessarily being attached

Reply:
It is the UserField's name, being read with the relationship defined just below the attribute. So how can it fail ? 

If someone deletes the UserField itself then the relationship will return nothing. Same like the $pirep->ident (or $flight->ident) attribute, in which we use the airline's code without checking the airline itself.

Do we need to check the related model's existence first while using `$this->relationship->...` in a model ?

Reply:
Because `$this->field` will then return null

Reply:
Yep, I was testing it too (by changing an airline's id to 9999 and see how it reacts)

So should we change all related attributes with checks like below;

`return isset($this->field) ? $this->field->name : null`

Reply:
Or maybe just `return optional($this->field)->name;`

Reply:
`optional()`

Reply:
Already done ... All of them